apps/site/util/downloadConstants.json

```diff
@@ -0,0 +1,40 @@
+{
+  "OperatingSystem": {
+    "WIN": "Windows",
+    "MAC": "macOS",
+    "LINUX": "Linux",
+    "AIX": "AIX",
+    "OTHER": "Other",
+    "LOADING": "N/A"
+  },
+
+  "InstallationMethod": {
+    "NVM": "nvm",
+    "FNM": "fnm",
+    "BREW": "Brew",
+    "CHOCO": "Chocolatey",
+    "DEVBOX": "Devbox",
+    "DOCKER": "Docker",
+    "N": "n",
+    "VOLTA": "Volta"
+  },
+  "PackageManager": {
+    "NPM": "npm",
+    "YARN": "Yarn",
+    "PNPM": "pnpm"
+  },
+  "UserPlatform": [
+    "arm64",
+    "armv7l",
+    "ppc64le",
+    "ppc64",
+    "s390x",
+    "ppc64",
+    "x64",
+    "x86"
+  ],
+
+  "UserOs": ["MAC", "WIN", "LINUX", "AIX", "OTHER"],
```

Comment:
This can be inferred from the keys of OperatingSystem, right?

Reply:
whoops! 

You are right!